7/Dockerfile.rhel7

```diff
@@ -1,51 +1,46 @@
-FROM rhel7
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+FROM rhscl/s2i-base-rhel7
 
-ENV SUMMARY="Red Hat LLVM Toolset 7 container image" \
-    DESCRIPTION="Platform for building applications using Red Hat LLVM Toolset 7"
+ENV LLVM_VERSION="4.0.x" \
+    SUMMARY="Platform for building projects with clang/llvm" \
+    DESCRIPTION="$SUMMARY"
 
-LABEL com.redhat.component="llvm-toolset-7-docker" \
-      name="devtools/llvm-toolset-7-rhel7" \
-      version="7" \
-      summary="$SUMMARY" \
+
+LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="LLVM Toolset 7"
+      io.k8s.display-name="$LLVM_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,llvm,clang" \
+      com.redhat.component="llvm-toolset-7-docker" \
+      name="devtools/llvm-toolset-7-rhel7" \
+      version="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
 
-# To use subscription inside container yum command has to be run first (before yum-config-manager)
-# https://access.redhat.com/solutions/1443553
-RUN yum repolist > /dev/null && \
-    yum-config-manager --disable \* &> /dev/null && \
-    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
-    yum-config-manager --enable rhel-server-rhscl-7-beta-rpms && \
+RUN yum install -y centos-release-scl-rh && \
```

Comment:
`centos-release-scl-rh` shouldn't be there IMHO.

Also `prepare-yum-repositories` script was introduced (https://github.com/sclorg/s2i-base-container/pull/149) to make setting up repositories easier, allow building with own repositories,...

So please use:
```
RUN prepare-yum-repositories rhel-server-rhscl-7-rpms rhel-7-server-devtools-rpms \
    INSTALL_PKGS= ...
```